Gemfile.lock

```diff
@@ -184,25 +184,25 @@ GEM
     turbolinks (5.2.1)
       turbolinks-source (~> 5.2)
     turbolinks-source (5.2.0)
-    tzinfo (1.2.6)
+    tzinfo (1.2.7)
       thread_safe (~> 0.1)
     uglifier (4.2.0)
       execjs (>= 0.3.0, < 3)
-    web-console (4.0.1)
+    web-console (4.0.4)
```

Comment:
### [web-console](https://github.com/rails/web-console)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 4.0.1 → 4.0.4

[_~~change-log~~, ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **11** commits. See the full changes on [the compare page](https://github.com/rails/web-console/compare/v4.0.1...v4.0.4).

These are the first 10 commits:
- (55d8485) [Increase timeout on paste](https://github.com/rails/web-console/commit/55d84856201a80353e4417408ccff88696334afa)
- (be84e64) [Merge pull request #285 from celvro/fix-paste](https://github.com/rails/web-console/commit/be84e6471c33bbbea17f6dfc9e5ed3c7c7c4bb57)
- (23e7ab1) [Release 4.0.2](https://github.com/rails/web-console/commit/23e7ab1d75bdbff4970dee3b55c42095ff1e8710)
- (b5ffa70) [Revise 4.0.0's changelog](https://github.com/rails/web-console/commit/b5ffa70f66e57e9ff8f04f03e23a0e1b482c0b64)
- (bcd7776) [Fix Content-Length for rack >= 2.1.0](https://github.com/rails/web-console/commit/bcd7776606b2ec9b3139897919e9df876e53fe10)
- (d9fd6d2) [Merge pull request #290 from p8/fix-for-rack-2.1.0](https://github.com/rails/web-console/commit/d9fd6d28ba206ff4415808f5b995c18574847181)
- (02f51e1) [Use config.web_console.allowed_ips and update related files](https://github.com/rails/web-console/commit/02f51e1818f5872e13a96b88f77f76c91de28649)
- (d5f289d) [Merge pull request #291 from JuanitoFatas/config_allowed_ips](https://github.com/rails/web-console/commit/d5f289dfef47c5c61fab88ec10e0106911ef3ccf)
- (b18a195) [Release 4.0.3](https://github.com/rails/web-console/commit/b18a195183f82a6480629a115895a6737de5c4a5)
- (fb48374) [Fix a crash on webrick with Rack 2.2.3](https://github.com/rails/web-console/commit/fb483743a6a2a4168cdc0b2e03f48fc393991b73)


</details>